ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -586,16 +641,23 @@ module.exports = function OpenlayersMap(
           this.updateGeometry(innerGeometry, options)
         })
       } else {
-        const feature = geometry.getSource().getFeatures()[0]
-        const geometryInstance = feature.getGeometry()
-        if (geometryInstance.constructor === Openlayers.geom.Point) {
-          let x = 39,
-            y = 40
-          if (options.size) {
-            x = options.size.x
-            y = options.size.y
-          }
-          geometry.setZIndex(options.isSelected ? 2 : 1)
+        const features = geometry.getSource().getFeatures()
+        features.forEach(feature =>
+          this.setGeometryStyle(geometry, options, feature)
+        )
+      }
+    },
+    setGeometryStyle(geometry, options, feature) {
+      const geometryInstance = feature.getGeometry()
+      if (geometryInstance.constructor === Openlayers.geom.Point) {
+        let x = 39,
+          y = 40
```

Comment:
Should these constants be defined somewhere? Are they only used here?

Reply:
I would think so, although I don't even know why those specific numbers were used.